tests/release/e2e-test-default-bundle.go

```diff
@@ -0,0 +1,240 @@
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates Role binding with a given name in managednamespace.", func() {
+			_, err := framework.CommonController.CreateRoleBinding("role-relase-service-account-binding", managedNamespace, "ServiceAccount", roleName, "Role", "role-m6-service-account", "rbac.authorization.k8s.io")
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates application with the given name in dev namespace.", func() {
+			_, err := framework.HasController.CreateHasApplication(applicationNameDefault, devNamespace)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("creates component with the given component name in dev namespace.", func() {
+			_, err := framework.HasController.CreateComponent(applicationNameDefault, componentName, devNamespace, gitSourceComponentUrl, "", containerImageUrl, "", "", false)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+	})
+
+	var _ = Describe("Post-release verification", func() {
+
+		It("verifies that in dev namespace will be created a PipelineRun.", func() {
+			Eventually(func() bool {
+				prList, err := framework.TektonController.ListAllPipelineRuns(devNamespace)
+				if err != nil || prList == nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return strings.Contains(prList.Items[0].Name, componentName)
+			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies that the PipelineRun in dev namespace succeeded.", func() {
+			Eventually(func() bool {
+				prList, err := framework.TektonController.ListAllPipelineRuns(devNamespace)
+				if prList == nil || err != nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies that in managed namespace will be created a PipelineRun.", func() {
+			Eventually(func() bool {
+				prList, err := framework.TektonController.ListAllPipelineRuns(managedNamespace)
+				if err != nil || prList == nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return strings.Contains(prList.Items[0].Name, "release")
+			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("verifies a PipelineRun started in managed namespace succeeded.", func() {
+			Eventually(func() bool {
+				prList, err := framework.TektonController.ListAllPipelineRuns(managedNamespace)
+				if prList == nil || err != nil || len(prList.Items) < 1 {
+					return false
+				}
+
+				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
+			}, releasePipelineRunCompletionTimeout, defaultInterval).Should(BeTrue())
+		})
+
+		It("tests a Release should have been created in the dev namespace and succeeded.", func() {
```

Comment:
Would this be the signal to validate that Release can succesfully trigger it's Release PipelineRun? So shouldn't this come before your managed services pipelinerun tests?

Reply:
the Release will be succeeded only after the release pipelierun finished and succeeded , 
so it would be verified only after Release PipelineRun is finished. 
see below example of Release Conditions while release PR is not done yet :

NAMESPACE                              NAME                    SNAPSHOT          SUCCEEDED   REASON    AGE
998b0055-da7c-492a-b897-aa7ae4acd63a   appstudio-tdlnp-25vcz   appstudio-tdlnp   Unknown     Running   93s

and when PR is done and succeeded like below : 
NAMESPACE                              NAME                    SNAPSHOT          SUCCEEDED   REASON      AGE
998b0055-da7c-492a-b897-aa7ae4acd63a   appstudio-tdlnp-25vcz   appstudio-tdlnp   True        Succeeded   95s

Reply:
Yes, the release would have been created before the other checks, but not succeeded until now